elastic/kibana
# [Guided onboarding] Implement search E2E guide

This PR completes the E2E search guide. Changes include adding the manual step completion popover and fixing the search experiences step url.

**How to test**
1. This is best to test on cloud. This PR has been deployed there. To reset the guide, use the example plugin and update the step ID to `add_data`.
2. Click "Create Elasticsearch index"
3. Select "Use a connector"
4. Create an index.
5. Use Dev Tools to index any document into the created index in step 4.
6. Step 1 should be marked as complete, and you can finish the guide.

Fixes https://github.com/elastic/kibana/issues/138998

https://user-images.githubusercontent.com/5226211/199627065-035b9d17-b5b3-4752-b843-188439820ebe.mp4

<img width="1460" alt="Screen Shot 2022-11-02 at 7 42 15 PM" src="https://user-images.githubusercontent.com/5226211/199622606-3106971e-2d59-4ed1-b31e-a52cae641c7f.png">
